edx_notifications/digest_timers.py

```diff
@@ -0,0 +1,49 @@
+"""
+Create and register a new NotificationCallbackTimerHandler
+"""
+import datetime
+from django.dispatch import receiver
+import pytz
+from edx_notifications.data import NotificationCallbackTimer
+from edx_notifications.signals import perform_timer_registrations
+from edx_notifications.stores.store import notification_store
+from edx_notifications.exceptions import ItemNotFoundError
+
+DAILY_DIGEST_TIMER_NAME = 'daily-digest-timer'
+MINUTES_IN_A_DAY = 24 * 60
+WEEKLY_DIGEST_TIMER_NAME = 'weekly-digest-timer'
+MINUTES_IN_A_WEEK = 7 * 24 * 60
+
+
+@receiver(perform_timer_registrations)
+def register_digest_timers(sender, **kwargs):  # pylint: disable=unused-argument
+    """
+    Register NotificationCallbackTimerHandler.
+    This will be called automatically on the Notification subsystem startup (because we are
+    receiving the 'perform_timer_registrations' signal)
+    """
+    store = notification_store()
+
+    try:
+        store.get_notification_timer(DAILY_DIGEST_TIMER_NAME)
+    except ItemNotFoundError:
+        daily_digest_timer = NotificationCallbackTimer(
+            name=DAILY_DIGEST_TIMER_NAME,
+            callback_at=datetime.datetime.now(pytz.UTC),
```

Comment:
Even though I just merged this in, I'm wondering if we need to set the callback_at (here and below) to be 00:00 UTC the following day. The way that it is now, the digest will be produced at the same time every day that the service first started up....

Can you change?